billing/data/space-diff.js

```diff
@@ -28,9 +27,8 @@ export const encode = input => {
   try {
     return {
       ok: {
-        pk: `${input.customer}#${input.provider}#${input.space}`,
+        pk: `${input.provider}#${input.space}`,
```

Comment:
can we please update `billing/tables/space-diff` comment on how `pk` is formated?